### ISSUE TITLE
body2 TextStyle is deprecated

### ISSUE THREAD

**Issue by Rex-Ferrer** [material-components/material-components-flutter-codelabs#184]
In MDC-102's Part 6(**Make a card collection**) and under the **Add product data** section, the deprecated **body2** TextStyle is used in the _buildGridCards(...) method. I recommend changing it to the **bodyText2** TextStyle.

### Current:
``` dart      
 Text(
    formatter.format(product.price),
    style: theme.textTheme.body2,
  ),
```

### Recommendation:
``` dart      
 Text(
    formatter.format(product.price),
    style: theme.textTheme.bodyText2,
  ),
```

**Comment by rami-a** [comment]
In the updated names the body texts swapped, so if it was `body2` before it should be `bodyText1` now.

**Comment by Renzo-Olivares** [comment]
Fixed by #189
